src/simulators/matrix_product_state/matrix_product_state_internal.cpp

```diff
@@ -349,6 +427,47 @@ void MPS::apply_diagonal_matrix(const AER::reg_t &qubits, const cvector_t &vmat)
   apply_matrix(qubits, diag_mat);
 }
 
+void MPS::centralize_qubits(MPS & new_MPS, const reg_t &qubits, 
```

Comment:
In the same spirit of the previous member function (`probabilities_vector`) I would change the signature of this member function to:
```
MPS MPS::centralize_qubits(const reg_t &qubits, uint_t &new_first, uint_t &new_last) const {
```
I have removed the `ordered` output argument because it's not being used outside of this function.

Reply:
removed.